### DISCUSSION TITLE
Avatarify Python Error

### DISCUSSION THREAD

**Discussion by IsmaelElHarras** [community/community#48414] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,
I open this discussion because recently I had a problem while trying to run the Avatarify program: https://github.com/technopremium/avatarify-python-youtube/blob/master/README.md

I have followed all the steps in the tutorial but it always blocks in the same place. I have tried with several computers and I have searched on the internet but I have not found the solution.

Is there anyone who has a clue? or someone who could help me to solve the problem 

Here is the error after launching the installation with the command: scripts\install_windows.bat

I'm stuck with this line which is just running in the void

![image](https://user-images.githubusercontent.com/79228694/221005355-389867c7-02fb-4f56-a241-c354ced62be0.png)

**Comment by IsmaelElHarras** [comment]
?

**Reply by professionnel1** [reply]
Hi did you solve this problem yet i have the same thing too

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.


_Also please note: The Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!_

**Comment by JustOptimize** [comment]
On what hardware are you running this? (Cpu, Gpu, Ram)

**Comment by justme220011** [comment]
please i am having the same problem please someone help please

**Comment by justme220011** [comment]
OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.

please this what i am getting i really need help on this please anyone who can help me out
